### PR TITLE
Fixed small bug in addNewTab functionality wrt updatePost datastore function

### PR DESCRIPTION
**Pull request by jindaln**

### CODE REVIEW

**Comment by v1shanker** [comment]
please rebase master

File: src/main/java/com/google/sps/servlets/RepUploadImageServlet.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ private String getBlobKeyUrl(HttpServletRequest request, String formInputElement
         if (blobKeys == null || blobKeys.isEmpty()) {
             return(Constants.DEFAULT_POLITICIAN_IMAGE_URL);
         } else {
-            return (SERVE_URL + blobKeys.get(0).getKeyString());
+            return(SERVE_URL + blobKeys.get(0).getKeyString());
```
Comment (epoms) [inline src/main/java/com/google/sps/servlets/RepUploadImageServlet.java:77]: keep the space

File: src/main/java/com/google/sps/servlets/NewPostServlet.java [hunk 2]
```diff
@@ -64,7 +64,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             commentId = DatastoreManager.insertCommentInDatastoreIfNonToxic(name, comment);
         } catch(ToxicCommentException e) {
             logger.error(e);
-            throw new ServletException("Error: " + e.getMessage(), e);
+            String redirect = (feedBool == true) ? "feed.html?name=" + URLEncoder.encode(repName) :
```
Comment (epoms) [inline src/main/java/com/google/sps/servlets/NewPostServlet.java:67]: you can just do feedBool ? <...>
also what is feedBool? could you rename it to something that actually describes what it is.

File: src/main/java/com/google/sps/data/Post.java [hunk 3]
```diff
@@ -116,8 +116,8 @@ public String toString() {
         @Override
         public int compare(Post a, Post b) {
             long currTime = System.currentTimeMillis(); 
-            double aRecency = a.getTimestamp()/currTime; 
-            double bRecency = b.getTimestamp()/currTime; 
+            double aRecency = (double)a.getTimestamp() / (double)currTime;
```
Comment (epoms) [inline src/main/java/com/google/sps/data/Post.java:119]: nit: put space after parentheses

**Pull request merged**